app/controllers/api/encryptables_controller.rb

```diff
@@ -126,7 +130,7 @@ def permitted_attrs
     permitted_attrs = self.class.permitted_attrs.deep_dup
 
     if model_class == Encryptable::File
-      permitted_attrs + [:filename, :credentials_id, :file]
+      permitted_attrs + [:filename, :credentials_id, :folder_id, :file]
```

Comment:
looks like folder_id is available for both model classes ... maybe add to default?